frontend/src/main/scala/bloop/data/ClientInfo.scala

```diff
@@ -43,25 +64,36 @@ object ClientInfo {
   final case class BspClientInfo(
       name: String,
       version: String,
-      bspVersion: String
+      bspVersion: String,
+      private val isConnected: () => Boolean
   ) extends ClientInfo {
+    val uniqueId: String = s"${this.name}-${UUIDUtil.randomUUID}"
```

Comment:
It will be just a random string from `java.util.Random`, we could use a connection timestamp instead but I've grown used to using a random UUID in base 64...